frontend/src/views/Profile.vue

```diff
@@ -39,21 +55,30 @@
         </v-skeleton-loader>
       </v-col>
     </content-card>
+
+    <dialog-change-mail-running :email-verification-key="emailVerificationKey" />
```

Comment:
I don't understand why you use two dialogs at all. I see no way to ever activate the `dialog-change-mail-running`, and the `dialog-change-mail` does a good job of showing the progress and completion status of the email change. Couldn't you remove this second dialog completely?